lua-resty-openssl-0.6.3-1.rockspec

```diff
@@ -1,5 +1,5 @@
 package = "lua-resty-openssl"
-version = "0.6.2-1"
+version = "0.6.3-1"
```

Comment:
let's keep the version and filename unchanged for now, i'll bump it up in release, otherwise below it's still pointing to the old tag in git

Reply:
After previous PR it was not updated. Therefore I bumped it up.
Revereted